### PR TITLE
Matenadaran nv

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Hello! Here I have created an institution record for the museum.

### CODE REVIEW

File: new/INS1021Marenadaran.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-11-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="museum" subtype="institution">
```
Comment (eu-genia) [inline new/INS1021Marenadaran.xml:47]: Q1322278

File: new/INS1021Marenadaran.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <placeName n="1" xml:lang="en">Mesrop Mashtots Research Institute of Ancient
```
Comment (karljonaskarlsson) [inline new/INS1021Marenadaran.xml:48]: Maybe it would be useful to add the name in Armenian? In case someone searches for it.

**Review by karljonaskarlsson — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/INS1021Marenadaran.xml (hunk 2 above)
Comment (eu-genia) [reply new/INS1021Marenadaran.xml:49]: sameAs needs `wd:` prefix, thank you!

File: new/INS1021Marenadaran.xml [hunk 3]
```diff
@@ -5,8 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title>Mesrop Mashtots Research Institute of Ancient Manuscripts
+                <title xml:lang="en">Mesrop Mashtots Research Institute of Ancient Manuscripts
```
Comment (eu-genia) [inline new/INS1021Marenadaran.xml:8]: the different names are not the titles of the file (titleStmt) but the variants of the placeName and must be moved there
(placeName is treated in the same way as the work title or persName with n="1" n="2" and/or `corresp`)
https://betamasaheft.eu/Guidelines/?id=places

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you

File: new/INS1021Marenadaran.xml (hunk 3 above)
Comment (nafisa-valieva) [reply new/INS1021Marenadaran.xml:8]: Thank you! Can you also explain me why I can´t see anything in Armenian in Oxygen, but I see it in Gitup history and what should I change to enable the reading?

Comment (eu-genia) [reply new/INS1021Marenadaran.xml]: You probably have to switch the main font to a Unicode font that has Armenian, I do not know which one you have, I use TITUS, it has Armenian. Ethiopic Unicode does not have Armenian or most other Oriental fonts.

**Review by eu-genia — CHANGES_REQUESTED** [review]
Thank you, just very minor things left

File: new/INS1021Marenadaran.xml (hunk 3 above)
Comment (eu-genia) [reply new/INS1021Marenadaran.xml]: image

Comment (nafisa-valieva) [reply new/INS1021Marenadaran.xml]: Thank you! I am trying to download TITUS but it seems not to be available. If you can direct me to the right page or send me the font via email, I would be very grateful!

File: new/INS1021Marenadaran.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <text>
+        <body>
+            <listPlace>
+                <place sameAs="Q1322278" type="museum" subtype="institution">
+                    <placeName n="1" xml:lang="en">Mesrop Mashtots Research Institute of Ancient
+                        Manuscripts</placeName>
+                    <placeName n="2" xml:lang="hy">Մատենադարան</placeName>
+                    <placeName corresp="#2" xml:lang="hy" type="normalized">Matenadaran</placeName>
+                    <settlement ref="LOC7457Yerevan"/>
+                    <country ref="wd:Q399"/>
```
[the remark this answers is not in the document]
Comment (nafisa-valieva) [reply new/INS1021Marenadaran.xml]: Thank you! following this discussion I see I am supposed to add bm ids to wikidata too, right? If this is the practice, I will ask Dorothea to show me how to do it once she is back. I do not think anyone besides her was actually doing it. (I was not).

File: new/INS1021Marenadaran.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-11-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place sameAs="Q1322278" type="museum" subtype="institution">
```
Comment (eu-genia) [inline new/INS1021Marenadaran.xml:47]: `wd: `is still missing here (must be sameAs="wd:Q1322278")

**Review by eu-genia — APPROVED** [review]

File: new/INS1021Marenadaran.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/INS1021Marenadaran.xml]: I was. Carsten too I believe. You just.need an account

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/INS1021Marenadaran.xml]: 1 log in to wikidata and go to the record
image

2 add BM Id in a statement 
image

that's it

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/INS1021Marenadaran.xml]: https://betamasaheft.eu/Guidelines/?id=places
image

**Pull request merged**